staging/src/k8s.io/apimachinery/pkg/util/proxy/upgradeaware.go

```diff
@@ -232,6 +232,7 @@ func (h *UpgradeAwareHandler) ServeHTTP(w http.ResponseWriter, req *http.Request
 	proxy.Transport = h.Transport
 	proxy.FlushInterval = h.FlushInterval
 	proxy.ErrorLog = log.New(noSuppressPanicError{}, "", log.LstdFlags)
+	proxy.ErrorHandler = h.Responder.Error
```

Comment:
what does this do?

Reply:
it passes errors to `h.Responser.Errror` (if provided) for evaluation.

Reply:
see https://golang.org/src/net/http/httputil/reverseproxy.go